express-post-json/index.js

```diff
@@ -0,0 +1,26 @@
+const express = require('express');
+const app = express();
+let nextId = 1;
+const grades = {};
+
+app.use(express.json());
+
+app.get('/api/grades', (req, res) => {
+  const gradesArray = [];
+  for (const property in grades) {
+    gradesArray.push(grades[property]);
+  }
+  res.json(gradesArray);
+});
+
+app.post('/api/grades', (req, res) => {
+  req.body.id = nextId;
+  res.send(req.body);
+
+  grades[''] = req.body;
```

Comment:
`grades[nextId]`.

If you assign every incoming object to `''`, then they will always erase each other.